### PR TITLE
Bug fix, docs, and tracer added to WKB wave packet test case

### LINKED ISSUE #260: Boundary conditions not set after turbulent diffusion of horizontal winds
In `turbulent_diffusion!`, the boundary conditions for `u` and `v` must be enforced after `turbulent_diffusion!(state, dt, U())` and `turbulent_diffusion!(state, dt, V())`. The diffusion is only applied to `u[i0:i1, j0:j1, k0:k1]` and `v[i0:i1, j0:j1, k0:k1]`. However, `turbulent_diffusion!(state, dt, W())` accesses `u` and `v` at the grid-cell indices `i0-1`, `j0-1`, and `k1+1`.

### PR DESCRIPTION
**Pull request by irmgard-knop**
This PR fixes #260. Additionally, I've fixed the two documentation issues in the "Numerics" section (see #192), as well as added a tracer to the function `wkb_wave_packet()`. That way we also cover the leading-order gravity-wave tracer fluxes in the tests.

### CODE REVIEW

**Review by Felix-Jochum — COMMENTED** [review]
Having included a tracer with corresponding gravity-wave fluxes in this example, don't you think it would make sense to also include some of that in the plot? Maybe you could even replace the momentum-flux plot - we already have one for the WKB-mountain-wave example. I don't know what would make more sense here - the file size shouldn't be an issue, I think it's more about what makes sense to the reader and isn't too much information crammed into a single example figure.

**Comment by irmgard-knop** [comment]
I'll check out the gravity-wave tracer flux convergence (`:dchidt0`).

**Comment by Felix-Jochum** [comment]
Before I forget to mention it: I just noticed that the label for the leading-order tracer impact is incorrect - there is a comma instead of a whitespace between name and unit (see [here](https://atmospheric-dynamics-guf.github.io/PinCFlow.jl/previews/PR261/examples/#WKB-wave-packet) for the plot and [here](https://github.com/Atmospheric-Dynamics-GUF/PinCFlow.jl/blob/5027927a36ead6df595389144e4f5713bac42b2f/src/Output/create_output.jl#L594) for where it needs to be fixed).

**Comment by irmgard-knop** [comment]
Thanks for pointing that out! I was also able to find a few more incorrect labels!

File: docs/src/examples.md [hunk 1]
```diff
@@ -739,7 +739,7 @@ end
 
 ```
 
-initializes an unresolved gravity-wave packet (i.e. one that is parameterized by MS-GWaM) in the stratosphere of a compressible atmosphere with two different lapse rates and visualizes the resulting zonal vertical-momentum flux after ten minutes integration time (see below). Like the wave-packet script discussed above, it constructs an auxiliary state and uses helper functions to satisfy the gravity-wave dispersion and polarization relations. Furthermore, it initializes a tracer field that increases linearly with altitude, including parameterized, leading-order gravity-wave tracer fluxes.
+initializes an unresolved gravity-wave packet (i.e. one that is parameterized by MS-GWaM) in the stratosphere of a compressible atmosphere with two different lapse rates Furthermore, it initializes a tracer field that increases linearly with altitude and visualizes the parameterized, leading-order gravity-wave tracer flux convergence after ten minutes integration time (see below). Like the wave-packet script discussed above, it constructs an auxiliary state and uses helper functions to satisfy the gravity-wave dispersion and polarization relations.
```
Comment (Felix-Jochum) [inline docs/src/examples.md]: A period is missing between "rates" and "Furthermore".

**Review by Felix-Jochum — CHANGES_REQUESTED** [review]

File: docs/src/examples.md (hunk 1 above)
Comment (irmgard-knop) [reply docs/src/examples.md]: Oof, missed that! Thanks!

**Review by Felix-Jochum — APPROVED** [review]
Thank you for these changes. Increasing test coverage is always a good idea.
